frontend/mariner2/handle_container.go

```diff
@@ -98,20 +100,20 @@ func buildImageConfig(ctx context.Context, spec *dalec.Spec, target string, clie
 	return &img, nil
 }
 
-func mergeSpecImage(spec *dalec.Spec, target string) *dalec.ImageConfig {
+func mergeSpecImage(spec *dalec.Spec, targetKey string) *dalec.ImageConfig {
 	var cfg dalec.ImageConfig
 
 	if spec.Image != nil {
 		cfg = *spec.Image
 	}
 
-	if i := spec.Targets[target].Image; i != nil {
+	if i := spec.Targets[targetKey].Image; i != nil {
 		if i.Entrypoint != "" {
-			cfg.Entrypoint = spec.Targets[target].Image.Entrypoint
+			cfg.Entrypoint = spec.Targets[targetKey].Image.Entrypoint
```

Comment:
```suggestion
			cfg.Entrypoint = i.Entrypoint
```

Reply:
This looks to be resolved primarily by the Windows PR which moved these functions around.